dwh/bq_to_es_projector/bq_to_es_projector.py

```diff
@@ -68,14 +52,45 @@ def make_es_client() -> Elasticsearch:
     return es
 
 
+def get_index_metadata(es_index_name: str) -> Tuple[str, str]:
+    """
+    Returns (index_prefix, key_field_name) for a given ES index name.
+    """
+    if es_index_name.startswith("contrast-summary"):
+        return "contrast", "contrast_id"
+    elif es_index_name.startswith("dataset-summary"):
+        return "dataset", "dataset_id"
+    elif es_index_name.startswith("target-summary"):
+        return "target", "perturbed_target_symbol"
+    elif es_index_name.startswith("gene-summary"):
+        return "gene", "gene"
+    elif es_index_name.startswith("landing-page-summary"):
+        return "landing-page", "summary"
+    else:
+        raise ValueError(f"Unknown index name: {es_index_name}")
+
```

Comment:
I would suggest to use endswith here - in another PR I've already used indices in the format YYYY-MM-DD_index_name and aliases on top of it. In the future it'll be easier to sort/update indices.